AutomatedTesting/AutomatedTesting_Dependencies.xml

```diff
@@ -1,5 +1,4 @@
 <EngineDependencies versionnumber="1.0.0">
     <Dependency path="libs/particles/preloadlibs.txt" optional="true" />
```

Comment:
If you're in here making changes, I don't think this is needed, hasn't the old particle system been totally removed?

Reply:
Well, it's optional.  Looks like there is a `libs/particles` folder there still.   No `preloadlibs.txt` file but a `milestone2particles.xml` file instead.  We can probably leave it this way for now.

Reply:
preloadlibs.txt was an optional dependency because in Lumberyard there was a hardcoded C++ attempt to load the particle systems in preloadlibs.txt in engine code. Because of that, it was marked as an optional dependency (it was fine if the file wasn't there) and tracked here. We had it in the project template for the same reason: If you create a new project, added a preloadlibs.txt to mark particle systems you wanted pre-loaded, that file needed to get included in asset bundles.

If we no longer have the old particle system and no longer have the call to preloadlibs.txt, then this wouldn't be useful anymore.